theme/lumo/vaadin-custom-field.html

```diff
@@ -1 +1,13 @@
 <link rel="import" href="../../src/vaadin-custom-field.html">
+<link rel="import" href="../../../vaadin-lumo-styles/mixins/required-field.html">
+
+<dom-module id="lumo-custom-field" theme-for="vaadin-custom-field">
```

Comment:
BTW the styles should be in `vaadin-custom-field-styles.html` and that file imported here. Not a blocker.

Reply:
Changed.